db/python/tables/bq/generic_bq_filter.py

```diff
@@ -0,0 +1,101 @@
+from datetime import datetime
+from enum import Enum
+from typing import Any
+
+from google.cloud import bigquery
+
+from db.python.utils import GenericFilter, T
+
+
+class GenericBQFilter(GenericFilter[T]):
+    """
+    Generic BigQuery filter is BQ specific filter class, based on GenericFilter
+    """
+
+    def to_sql(
```

Comment:
NGL, I find this mentally hard to review - not because of your code, but probably I'm fatigued. I think it looks all pre fine - can you prioritise this in the testing?

Reply:
Yes, this has bunch of unit tests in the next coming PR after this.